roles/ansible-keylime-tpm20/tasks/keylime.yml

```diff
@@ -15,6 +15,31 @@
     mode: 0644
     remote_src: yes
 
+- name: Create /etc/keylime for new-format config files
+  ansible.builtin.file:
+    path: /etc/keylime
+    state: directory
+    mode: 0755
+
+- name: Convert keylime.conf to new config format
+  shell: "/usr/bin/python3 convert_config.py"
+  args:
+    chdir: /root/keylime/scripts
+
+- name: Copy converted config files to /etc/keylime
+  copy:
+    src: "{{ item }}"
+    dest: /etc/keylime
+    mode: 0644
+    remote_src: yes
+  loop:
+    - /root/keylime/scripts/agent.conf
```

Comment:
Instead of making each variable in the loop inclue the "/root/keylime/scripts/" prefix, just name them what is different: ie, "agent", "verifier", "tenant", etc. Then in the `src` include the repeated parts like "/root/keylime/scripts/{{ item }}.conf"

Reply:
@mpeters Fixed!